setup.py

```diff
@@ -20,7 +20,8 @@
 
 
 DEPENDENCIES = (
-    "cachetools>=2.0.0,<5.0",
+    'cachetools>=2.0.0,<5.0; python_version < "3.7"',
+    'cachetools>=2.0.0,<6.0; python_version >= "3.7"',
```

Comment:
```suggestion
    'cachetools>=2.0.0,<6.0',
```
That shouldn't be necessary. Pip can automatically resolve the compatible version. I.e. pip will choose `<5.0` for Python 3.6.

Reply:
Ah sorry, I lost track of this PR. I've condensed this down to one pin as suggested.